ci/jenkinsfiles/common-lib.groovy

```diff
@@ -155,6 +154,17 @@ String getResourceName(String resource, String namespace) {
     ).trim()
 }
 
+String getRetentionMode() {
+  String retentionMode = 'compliance' //Default retention mode
```

Comment:
Does this mean `Compliance mode` is the default one and we will only switch between `Compliance` and `Governance` modes?

Reply:
Indeed, I'm supposing that the compliance mode will be the default one.
The end-user will be able to switch on governance using GitHub label.
perhaps I'm wrong.